core/web-assets/src/main/assets/js/apps/onms-endpoints/grafana/grafana-modal.html

```diff
@@ -0,0 +1,95 @@
+<style type="text/css">
+  .rotate {
+    -webkit-animation: rotation 2s infinite linear;
+  }
+
+  @-webkit-keyframes rotation {
+    from {
+      -webkit-transform: rotate(0deg);
+    }
+    to {
+      -webkit-transform: rotate(359deg);
+    }
+  }
+</style>
+
+
+<div id="endpointModal">
+  <div class="modal-header">
+    <h5 class="modal-title">{{endpoint.id === undefined ? "Add" : "Edit"}} Grafana Endpoint</h5>
+  </div>
+  <div class="modal-body">
+    <form name="endpointForm" novalidate>
+      <div class="form-group">
+        <div ng-show="error.entity" class="form-text text-danger">{{error.entity}}</div>
+      </div>
+      <div class="form-group form-row">
+        <label class="col-form-label" for="endpoint.uid">Grafana ID</label>
```

Comment:
This field is called "Unique ID" in the table, but it's called "Grafana ID" in the modal. Is that intentional?

Reply:
Oh, good catch. Originally the page was supposed to hold all endpoints and differentiate via a type column. This is now gone, so the column should be called `Grafana ID` instead.